### DISCUSSION TITLE
How to downgrade without needing the extension

### DISCUSSION THREAD

**Discussion by funkymokey** [3kh0/ext-remover#548] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
is there any possible way? my chromebook got upgraded and i cant use my computer because im grounded or a file of grunt recovery 105

**Comment by obkduishinijf** [comment, marked as answer]
don't think that you can downgrade w/o the listed stuff. 🤷

**Comment by amelia-bartlett** [comment]
> is there any possible way? my chromebook got upgraded and i cant use my computer because im grounded or a file of grunt recovery 105

you can use the `dd` command on MacOS or Linux (this is how I do it since the extension doesn't support Linux)
